HW1/P5/P5.txt

```diff
@@ -0,0 +1,9 @@
+# Your discussion here
+For Kevin to Harvard, I was able to run bfs on my local machine in 2 minutes. But I am only able to run bfs from Harvard to Kevin on AWS.
+Using 5 instances, I was able to get the result in around 15 minutes. I don't think my code is parallized enough, there are a few collects
```

Comment:
Yes, I think the collect is definitely one of the issues. Try to think about how to approach this program in a parallel way, not just in a serial way.